src/main/java/com/coveros/selenified/application/Check.java

```diff
@@ -368,17 +360,17 @@ default String checkAlertMatches(String expectedAlertPattern, double waitFor, do
      * @param timeTook - the amount of time it took for wait for something (assuming we had to wait)
      * @return Boolean: whether or not the confirmation is actually present
      */
-    default boolean checkConfirmationPresent(double waitFor, double timeTook) {
+    boolean checkConfirmationPresent(double waitFor, double timeTook) {
         // check for the object to be present
         String confirmation;
-        boolean isConfirmationPresent = getApp().is().confirmationPresent();
+        boolean isConfirmationPresent = this.app.is().confirmationPresent();
         if (isConfirmationPresent) {
-            confirmation = getApp().get().confirmation();
+            confirmation = this.app.get().confirmation();
         } else {
-            getReporter().fail("to find a confirmation on the page", waitFor, NO_CONFIRMATION, timeTook);
+            this.reporter.fail("to find a confirmation on the page", waitFor, NO_CONFIRMATION, timeTook);
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "to find a confirmation on the page" 4 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/sonar/coding_rules#rule_key=squid%3AS1192)